drupal/acquia/acquia_code_deploy.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# This script creates a new tag and deploys it to either HUD8 or Academic Departments Acquia Prod environments.
+
+# $ sh ~/Sites/_hal/drupal/acquia/acquia_code_deploy.sh
+#
+# Notes:
+# - See README.md for detailed instructions.
+#
+# Dependencies:
+# - Drush: https://www.drush.org/
+#
+# Paramaters:
+# - Acquia Env | The environment to create the site on
+# - E-mail | Acquia E-mail
+# - Private Key | Acquia private key
+
+source ~/Sites/_hal/hal_config.txt
+
+#Select desired Acquia Enviroment
+echo "Select a Acquia Enviroment"
+TO_ACQUIA_ENVS=( "@hud8" "@academicdepartments" )
+select ACQUIA_ENV in "${TO_ACQUIA_ENVS[@]}"; do
+  if [[ -z "$" ]]; then
+    printf '"%s" is not a valid choice\n' "$REPLY" >&2
+  else
+    break
+  fi
+done
+
+#select  dev docroot folder
+if [ $ACQUIA_ENV = "@hud8" ]
+then
+  cd ${LOCAL_HOWARD_D8_FOLDERS[0]}/docroot/
```

Comment:
why the /docroot folder ? shouldnt git actions be performed at root, or am I missing something?